code/modules/vehicles/armored/interiors/chairs.dm

```diff
@@ -6,119 +6,84 @@
 	resistance_flags = RESIST_ALL
 	dir = EAST
 
-
-/obj/structure/bed/chair/vehicle_driver_seat
+/obj/structure/bed/chair/vehicle_crew
 	name = "driver seat"
 	icon = 'icons/obj/armored/3x3/tank_interior.dmi'
 	icon_state = "vehicle_chair"
 	resistance_flags = RESIST_ALL
 	dir = EAST
 	///owner of this object, assigned during interior linkage
 	var/obj/vehicle/sealed/armored/owner
+	///The skill required to man this chair
+	var/skill_req = SKILL_LARGE_VEHICLE_VETERAN
+	///modifier to view range when in this chair. Not additive mod
+	var/vis_range_mod
```

Comment:
why is this both on the chair and on the tank given we have a reference? I dont see the use, especially if someone does something like add a module like zoom that changes it

Reply:
Uh, because I had it on chairs first then had the idea of adding it to the vehicle after so I wouldn't have to have entirely new interiors just for chair child types. Updated.